abci/abci.go

```diff
@@ -23,7 +19,8 @@ type (
 		IsAuctionTx(tx sdk.Tx) (bool, error)
 	}
 
-	ProposalHandler struct {
+	//nolint:all
+	ABCIHandler struct {
```

Comment:
Added this here for purposes of keeping the name `ABCIHandler` since go lint was not happy.

Reply:
We should have two handlers:

* `ProposalHandler`
* `VoteExtensionHandler`